source/m2mnsdlinterface.cpp

```diff
@@ -74,7 +74,7 @@ M2MNsdlInterface::M2MNsdlInterface(M2MNsdlObserver &observer, M2MConnectionHandl
     // and free functions in structure and used functions for sending
     // and receiving purposes.
     _nsdl_handle = sn_nsdl_init(&(__nsdl_c_send_to_server), &(__nsdl_c_received_from_server),
-                 &(__nsdl_c_memory_alloc), &(__nsdl_c_memory_free));
+                 &(memory_alloc), &(memory_free));
```

Comment:
In fact signatures seemed to be same:
-void* __nsdl_c_memory_alloc(uint16_t size)

Both used uint16_t type size parameter.